packages/trpc/server/routers/viewer/bookings/__tests__/editLocation.handler.test.ts

```diff
@@ -1,18 +1,10 @@
-import {
-  createBookingScenario,
-  getOrganizer,
-  TestData,
-  getScenarioData,
-} from "@calcom/web/test/utils/bookingScenario/bookingScenario";
-import { getZoomAppCredential } from "@calcom/web/test/utils/bookingScenario/bookingScenario";
-import { expectSuccesfulLocationChangeEmails } from "@calcom/web/test/utils/bookingScenario/expects";
```

Comment:
This caused a huge amount of issues and in fact makes this entire test invalid, needless to say trpc should not depend on calcom/web